images/defaultbackend/configs/latest.apko.yaml

```diff
@@ -0,0 +1,30 @@
+contents:
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - ingress-default-backend
```

Comment:
Maybe a dumb question — should the image's name match this package name?

Reply:
maybe. 
and also maybe i did the wrong naming in the apk package

should we update there?

Reply:
renamed